### DISCUSSION TITLE
Classical algorithms in liboqs

### DISCUSSION THREAD

**Discussion by T-449** [open-quantum-safe/liboqs#1846] [Q&A] [answered]
What classical algorithms does liboqs support(if any)?

**Comment by cothan** [comment, marked as answer]
None. Liboqs are about Post-Quantum public key algorithms. 
- kem.h exposes KEM algorithms.
- sig.h and sig_stfl.h expose digital signature algorithms.

SHA2 and SHA3/SHAKE are supported in liboqs, they are classical symmetric algorithms if you count them.
